pkgs/by-name/op/openasar/package.nix

```diff
@@ -1,14 +1,14 @@
-{ lib, stdenv, fetchFromGitHub, nodejs, bash, asar, unzip }:
+{ lib, stdenv, fetchFromGitHub, unstableGitUpdater, nodejs, bash, asar, unzip }:
 
 stdenv.mkDerivation rec {
```

Comment:
I would propose using `(finalAttrs: {` instead

Reply:
I normally do that too, but I didn't really clean it up, only moved it.

Reply:
Cleaned up now